packages/enhanced/src/lib/utils/semver.js

```diff
@@ -0,0 +1,487 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Tobias Koppers @sokra
+*/
+
+'use strict';
+
+/** @typedef {import("../RuntimeTemplate")} RuntimeTemplate */
+/** @typedef {(string|number|undefined|[])[]} SemVerRange */
+
+/**
+ * @param {string} str version string
+ * @returns {(string|number|undefined|[])[]} parsed version
+ */
+const parseVersion = (str) => {
```

Comment:
This file might replace the semver in the runtime

Reply:
yes i hope so, much lighter than the package